hanabi.py

```diff
@@ -144,6 +133,79 @@ def evaluate(self, turn: Turn):
         self.turns.append(turn)
         self.turn_index += 1
 
+    def _clue_turn(self, turn: Clue):
+        target = turn.target
```

Comment:
Far more sensible ty. Optional: maybe replace the argument name with `clue: Clue` instead